source/macos/AdaptiveCards/AdaptiveCardsTests/Renderers/InputTimeRendererTests.swift

```diff
@@ -45,6 +45,42 @@ class InputTimeRendererTest: XCTestCase {
         let inputTimeField = renderTimeInput()
         XCTAssertEqual(inputTimeField.maxDateValue, "16:00")
     }
+    
+    func testClearsText() {
+        let val: String = "16:23:30"
+        inputTime = .make(value: val)
+
+        let inputTimeField = renderTimeInput()
+        inputTimeField.clearButton.performClick()
+        XCTAssertEqual(inputTimeField.dateValue, "")
+        XCTAssertTrue(inputTimeField.clearButton.isHidden)
+    }
+    
+    func testClearButtonHiddenWithPlaceholder() {
+        let placeholderString: String = "Sample Placeholder"
+        inputTime = .make(placeholder: placeholderString)
+
+        let inputTimeField = renderTimeInput()
+        inputTimeField.clearButton.performClick()
+        XCTAssertTrue(inputTimeField.clearButton.isHidden)
+    }
+
+    func testClearButtonHiddenWithNoText() {
+        inputTime = .make()
+
+        let inputTimeField = renderTimeInput()
+        inputTimeField.clearButton.performClick()
+        XCTAssertTrue(inputTimeField.clearButton.isHidden)
```

Comment:
```suggestion

        let inputTimeField = renderTimeInput()
        XCTAssertTrue(inputTimeField.clearButton.isHidden)
    }

    func testClearButtonHiddenWithNoText() {
        inputTime = .make()

        let inputTimeField = renderTimeInput()
        XCTAssertTrue(inputTimeField.clearButton.isHidden)
```

Use `performClick()` only in places where it's required to simulate a click.
These cases doesn't make sense to have click events.

Reply:
Sure will do thanks!